.travis.yml

```diff
@@ -61,17 +61,17 @@ jobs:
     python: 3.6
     name: 'Without Deps 3.6'
     script:
-    # test without cython, ujson and email-validator
-    - pip uninstall -y ujson email-validator
+    # test without cython, ujson, email-validator, and typing-extensions
```

Comment:
I think you can remove all these comments, this one is wrong and they're tautologous